---
layout: default
title: "Research"
classes:  landing
permalink: /research/
---


<section class="page-section full-page-layout fade-in">
    <div class="sidebar-image">
      <img src="{{ '/assets/images/Thesis_cover.png' | relative_url }}" alt="Vertical Sidebar Image" />
    </div>
    <div class="main-content">
    <div style="padding-top: 500px;"></div>
        We are a data-driven research group focusing on uncovering the missed layer of gene expression - RNA translation, beyond what the current genome annotation tells us. Our work focuses on discovering non-canonical proteins and hidden translation events that play critical roles in human disease. Using high-throughput data and computational approaches, we aim to rewrite the annotations of translation regions throughout the genome and using comparative translatomics identify novel biomarkers or targets in disease.


      <h2>Discovering unannotated open reading frames</h2>
      Emerging research across the field have revealed a substantial portion of gene expression activity that was previously missed. Many human transcripts produce multiple distinct translated products, leading to at least a 40% increase in identified translated regions including 1000s found in long non-coding RNAs. Such translated Open Reading Frames (ORFs) encoded in the genome have been previously unannotated in traditional databases such as GENCODE. We developed a human bodymap of RNA translation using Ribosome profiling of various human primary cells and tissues and developed data-driven translation signature scores to identify a high-quality set of ORFs that have translation signature profiles identical to known proteins. We are currently working as part of a consortia to standardize these annotations and make them available to the research community. 
      <h2>Translation dysregulation in disease</h2>
      Previously, we have shown detection of widespread gene expression changes due to translational regulation in a well-known pathway in heart scarring (TGFB1 signalling pathway) that were previously missed in traditional RNA-based target discovery screens. This has led to the understanding that not only we have missed dysregulated gene targets but misinterpreted several targets when studies were only based on RNA-seq data. Pinpointing targets that are translational regulated also allows to identify RNA-binding protein as regulators of translation. <br><br><br>

     Overall, these algorithms developed provide a powerful platform to gain insights on the translatome in any given system, as demonstrated by their work in fibrosis, inflammation, and cancer, paving the way for future breakthroughs in RNA biology.

    </div>
    <br>
    <br>
    <br>
    <br> 
</section>
